spec/services/csv_imports/gias_import_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe CsvImports::GiasImport do
+  subject { described_class.call("spec/fixtures/test_gias_import.csv") }
+
+  it "upserts the correct schools" do
+    expect { subject }.to change(GiasSchool, :count).from(0).to(1)
+  end
```

Comment:
Is it worth also testing that it handles the 'update' path rather than only 'insert'?

This test only proves that we insert new schools.